scripts/go-build.sh

```diff
@@ -49,8 +49,9 @@ fi
 ldflags=(-s -w "${x_defs[@]}")
 
 [[ -n "${GOOS}" ]] || die "GOOS must be set"
+[[ -n "${GOARCH}" ]] || die "GOARCH must be set"
 bin_name="$(basename "$main_srcdir")"
-output_file="bin/${GOOS}/${bin_name}"
+output_file="bin/${GOOS}/${GOARCH}/${bin_name}"
```

Comment:
Could change this to be part of the file if that makes more sense.

```suggestion
output_file="bin/${GOOS}/${bin_name}-${GOARCH}"
```

Reply:
I think we can change it